bin/stop_be.sh

```diff
@@ -76,20 +76,14 @@ if [ -f $pidfile ]; then
         exit 1
     fi
 
-    if kill -0 $pid; then
-        if kill -${sig} $pid > /dev/null 2>&1; then
-            echo "stop $pidcomm using signal $sig, and remove pid file. "
-            rm $pidfile
-            exit 0
-        else
+    if kill -0 $pid >/dev/null 2>&1; then
+        kill -${sig} $pid > /dev/null 2>&1
+        if [ $? -ne 0 ]; then
```

Comment:
why remove all the alert messages?

Reply:
It's nonsense and useless. The purpose of the stop_be.sh is to stop the be and remove the $pidfile. The $pidfile will be removed if kill success. What's the alert message used for?